libevm/tooling/release/README.md

```diff
@@ -0,0 +1,26 @@
+# Release
+
+In the following, we create a release candidate tag `v1.13.14-0.2.0.rc.4`.
+
+1. Create two branches, usually from the `main` branch:
+
+    ```bash
+    git fetch origin main:main
+    git checkout main
+    git checkout -b release/v1.13.14-0.2.0.rc.4
+    git push -u origin release/v1.13.14-0.2.0.rc.4
+    git checkout -b myusername/release/v1.13.14-0.2.0.rc.4
+    git push -u origin myusername/release/v1.13.14-0.2.0.rc.4
+    ```
+
+    The `myusername/release/v1.13.14-0.2.0.rc.4` branch will be used to add "release modifications" and will target the branch `release/v1.13.14-0.2.0.rc.4` in a pull request.
+1. Run script `./cherrypick.sh` which cherry picks all Geth commits listed in [cherrypicks](cherrypicks)
+    - you may have to resolve conflicts
+1. Modify [params/version.libevm.go](/params/version.libevm.go)
```

Comment:
It's useful to have this as the first step but NOT have anything committed. That way the version string is automatically generated for naming the release branch.

Reply:
> That way the version string is automatically generated for naming the release branch.

Through a test right?
I think it's best to keep it as it is, and have the CI check on the version string humanly typed, to avoid any bad copy-paste surprises.